recipe/bld.bat

```diff
@@ -1,4 +1,11 @@
-move bin\* %LIBRARY_BIN%\
+REM Move only desired files as the downloaded package contains even Qt5 dlls
+move bin\cmake.exe %LIBRARY_BIN%\
+move bin\cmcldeps.exe %LIBRARY_BIN%\
+move bin\cpack.exe %LIBRARY_BIN%\
+move bin\ctest.exe %LIBRARY_BIN%\
+move bin\msvcp120.dll %LIBRARY_BIN%\
```

Comment:
This is provided in `vs2013_runtime` package from defaults. https://anaconda.org/anaconda/vs2013_runtime